thumbor/filters/watermark.py

```diff
@@ -27,7 +28,7 @@ def detect_and_get_ratio_position(pos, length):
             return pos
 
         sign, ratio = match.groups()
-        pos = "{sign}{pos}".format(sign=sign, pos=int(round(length * float(ratio) / 100, 0)))
+        pos = "{sign}{pos}".format(sign=sign, pos=int(round(length * float(ratio) / 100 + 1e-5, 0)))
```

Comment:
Is `float` necessary here?

Reply:
Presumably yes, since `ratio` is a string. Or maybe you’re asking if this could this be done with integer arithmetic: `(length * int(ratio) + 50) // 100`?

Reply:
Had to keep `float` because `ratio` is indeed string, but managed to remove outer int.